docs/submission/clinical-data-validation-rules.md

```diff
@@ -6,8 +6,52 @@ platform_key: DOCS_CLINICAL_VALIDATION_RULES
 
 ![Reminder Banner](/assets/submission/banner-reminder.svg)
 
+## Clinical Data Encoding Rules
+
+### Identifier fields
+
+The data dictionary contains certain data elements regarded as "identifiers". These fields have an ![ID](/assets/submission/dictionary-id.svg) descriptor in the data dictionary and include:
+
+- Donor: `submitter_donor_id`
+- Specimen: `submitter_specimen_id`
+- Sample: `submitter_sample_id` 
+- Primary Diagnosis: `submitter_primary_diagnosis_id`
+- Treatment: `submitter_treatment_id`
+- Follow Up: `submitter_follow_up_id`
+
+These fields must to be coded specifically for ICGC-ARGO purposes using the following rules:
```

Comment:
@rosibaj we haven't been using a dash ever, but I did see some demos where Christina did this too. Do we want to change it so we always use a dash? Or just accept both ways of writing ICGC ARGO?

Reply:
lets be consistent i our writing.  lets go no dash as thats mostly everywhere.

its just natural for people to want to put a dash.

Reply:
yes I agree..especially since it's in the url. ok sounds good, Hardeep can you remove the dash please?

Reply:
Sure no problem, I'll remove the dash.